kubernetes/cmsweb/scripts/test_k8s_aps.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+##H Usage: test_k8s_aps.sh <base_url> <url_list_file>
+##H
+
+if [ "$1" == "-h" ] || [ "$1" == "-help" ] || [ "$1" == "--help" ] || [ "$1" == "help" ] || [ "$1" == "" ]; then
+    grep "##H" $0 | sed -e "s,##H,,g"
+    exit 1
+fi
+
+# Check if required arguments are provided
+if [ $# -ne 2 ]; then
+    echo "Error: You need to provide <base_url> and <url_list_file>"
+    grep "##H" $0 | sed -e "s,##H,,g"
+    exit 1
+fi
+
+X509_USER_KEY=$HOME/.globus/userkey.pem
+X509_USER_CERT=$HOME/.globus/usercert.pem
+
```

Comment:
Aroosha, please add check for X509_USER_PROXY env, if it is set we should use it instead of X509_USER_KEY/CERT since it is a proxy rather a user certificate. We may need to test both from time to time.